src/main/java/com/liferay/faces/util/product/internal/ProductBaseImpl.java

```diff
@@ -39,7 +39,7 @@
 	protected int patchVersion;
 	protected String stringValue;
 	protected String title;
-	protected String version;
+	protected String version = majorVersion + "." + minorVersion + "." + patchVersion;
```

Comment:
OK, I did a subsequent commit changing it to the String literal "0.0.0" because that looks more intentional to me.